### PR TITLE
add can-ajax docs

### PR DESCRIPTION
**Pull request by cherifGsoul**
this solves https://github.com/canjs/can-ajax/issues/9

### CODE REVIEW

**Comment by cherifGsoul** [comment]
@phillipskevin the requested changes are commited thank you for checking :)

**Comment by cherifGsoul** [comment]
@phillipskevin the tests fail because of a time out error, all tests pass localy !!

**Comment by cherifGsoul** [comment]
I think the test folder in ```can-ajax``` npm module is needed the tests needs them

File: test/test-production.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ require('can-symbol/can-symbol-test');
 require('can-reflect/can-reflect-test');
 require('can-reflect-promise/test/can-reflect-promise_test');
 require('can-attribute-encoder');
+require('can-ajax');
```
Comment (phillipskevin) [inline test/test-production.js]: This should be the test file. Also.. the one above this is wrong, should import the tests for `can-attribtue-encoder` if you want to fix that up also.

File: package.json [hunk 2]
```diff
@@ -65,6 +66,7 @@
     "can-jquery": "3.2.1",
     "can-kefir": "0.2.1",
     "can-list": "3.2.1",
+    "can-make-map": "^1.0.0",
```
Comment (phillipskevin) [inline package.json]: This should be a devDependency.

Comment (cherifGsoul) [reply package.json]: yes

**Pull request merged**